frontend/src/pages/index.tsx

```diff
@@ -1,3 +1,30 @@
 export default function Index() {
-	return <h1>signUp/signIN page</h1>;
+	return (
+		// <!-- This is button of signUp/signIn component -->
+		<div className="bg-white-700 py-9 px-9">
+			<div className="min-h-screen  sm:flex sm:flex-row  justify-center bg-gray-900 p-5 sm:p-60 rounded-3xl shadow-xl">
```

Comment:
Remove `min-h-screen`
and change  `sm:p-60`  to  `sm:p-10`

Reply:
I changed the style please check again.